Fleshing out community landing page to address /website/issues/307 (#357)
ipfs/docs: content/community/_index.md

headless: true
opened: true
---

> The IPFS community is as decentralized and distributed as IPFS itself — our community members are globally located, active online and in person, and using and building on IPFS in a virtually endless number of ways. You can get more information about meetings, events, apps people are building, and more using the resources listed here.

Reviewer: ```suggestion
The IPFS community is as decentralized and distributed as IPFS itself. Our community members are globally located, active both online and in person, and use IPFS in countless different ways. You can learn about meetings, events, apps people are building, and more with the resources listed here.
```

## Discussion/Support Forum

Join the conversation on the [https://discuss.ipfs.io/](IPFS Forums) for everything from technical questions, project brainstorming, talk on IPFS use cases and more.

## Research Discussion

The [IPFS Collaborative Notebook for Research](https://github.com/ipfs/notes) repo tracks discussions on topics related to IPFS research, ideas and proposals for new systems/features, and other items that haven't yet been tracked into a specific project.

## Events Calendar

Find all of the open community video calls for the IPFS project in the [IPFS Community events calendar](https://calendar.google.com/calendar/embed?src=ipfs.io_eal36ugu5e75s207gfjcu0ae84@group.calendar.google.com).

## Community Tools

> Join the discussion and documentation on IPFS community practices in the [community tools repo](https://github.com/ipfs/community/). If you're interested in starting, helping with or expanding IPFS awareness in your local geographical or online community, you can find tools and info here.

Reviewer: Not sure what "join the discussion and documentation" means here. Also lmk if my suggestion for the last sentence messed with its meaning!

```suggestion
Join the discussion and documentation on IPFS community practices in the [community tools repo](https://github.com/ipfs/community/). If you want to start or get involved with a community chapter to help expand IPFS awareness in your local geographical or online community, here's where to look.
```

## Project Management

> Keep abreast on IPFS team planning, management and coordination issues in the [IPFS team management repo](https://github.com/ipfs/team-mgmt).

Reviewer: ```suggestion
Keep abreast of IPFS team planning, management and coordination issues in the [IPFS team management repo](https://github.com/ipfs/team-mgmt).
```

## Blog

> Explore the latest news, events and other happenings in the IPFS multiverse in the official [https://ipfs.io/blog/](IPFS Blog).

Reviewer: ```suggestion
Explore the latest news, events and other happenings in the IPFS multiverse on the official [https://ipfs.io/blog/](IPFS Blog).
```

## IPFS YouTube channel

The [IPFS YouTube channel](https://www.youtube.com/channel/UCdjsUXJ3QawK4O5L1kqqsew) is home to a wealth of information about the IPFS project — everything from IPFS Camp course content to working group meeting archives — so you can explore playlists and subscribe to ones that interest and inform you.

## Twitter

Get your IPFS news in tweet-sized bites. Follow these accounts for the latest:

- [@IPFSbot](https://twitter.com/IPFSbot) for news and other updates
- [@IPFSevents](https://twitter.com/ipfsevents) for info on IPFS-related events
- [@ProtoSchool](https://twitter.com/protoschool) for updates on ProtoSchool chapters and tutorials

## Community projects on Awesome IPFS

> Get inspired by [Awesome IPFS](https://awesome.ipfs.io/), a catalog of community-built projects using IPFS for everything from image sharing to decentralized archiving. Have a project of your own to add to Awesome IPFS? Make a PR in the [repo](https://github.com/ipfs/awesome-ipfs)!

Reviewer: ```suggestion
Get inspired by [Awesome IPFS](https://awesome.ipfs.io/), a catalog of community-built projects using IPFS for everything from image sharing to decentralized archiving. Have a project of your own to add to Awesome IPFS? [Make a PR](https://github.com/ipfs/awesome-ipfs) in the repo!
```
